spec/book_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rspec'
```

Comment:
[Optional] Is it required to import `rspec` into our file? I don't think it is. If it's not required, it'll be nicer not to.